### ISSUE TITLE
Dialogs don't work on iOS on 3.1.14+

### ISSUE THREAD

**Issue by kmmccafferty96** [Akylas/nativescript-material-components#133]
### Which platform(s) does your issue occur on?

iPhone 11 Simulator running iOS 13.4

### Please, provide the following version numbers that your issue occurs with:

- CLI: 6.5.0
- Cross-platform modules: 6.5.1
- iOS Runtime: 6.5.0
- Plugins:
```
    "dependencies": {
        "@angular/animations": "~8.2.0",
        "@angular/common": "~8.2.0",
        "@angular/compiler": "~8.2.0",
        "@angular/core": "~8.2.0",
        "@angular/forms": "~8.2.0",
        "@angular/platform-browser": "~8.2.0",
        "@angular/platform-browser-dynamic": "~8.2.0",
        "@angular/router": "~8.2.0",
        "@nativescript/theme": "^2.3.3",
        "moment": "^2.24.0",
        "nativescript-angular": "^8.21.0",
        "nativescript-bottomnavigation": "^1.0.5",
        "nativescript-datetimepicker": "^1.2.1",
        "nativescript-image": "3.0.1",
        "nativescript-material-activityindicator": "^3.1.16",
        "nativescript-material-bottomnavigationbar": "^3.1.16",
        "nativescript-material-button": "^3.1.16",
        "nativescript-material-cardview": "^3.1.16",
        "nativescript-material-core": "^3.1.16",
        "nativescript-material-dialogs": "^3.1.16",
        "nativescript-material-snackbar": "^3.1.16",
        "nativescript-material-textfield": "^3.1.16",
        "nativescript-ui-sidedrawer": "^8.0.1",
        "reflect-metadata": "~0.1.12",
        "rxjs": "^6.5.5",
        "tns-core-modules": "^6.5.1",
        "zone.js": "^0.9.1"
    },
    "devDependencies": {
        "@angular/compiler-cli": "~8.2.0",
        "@ngtools/webpack": "~8.2.2",
        "nativescript-dev-webpack": "^1.5.1",
        "node-sass": "^4.13.1",
        "prettier": "^2.0.4",
        "tslint": "^6.1.1",
        "typescript": "~3.5.3"
    },
```

### Please, tell us how to recreate the issue in as much detail as possible. 

Whenever I call any of the methods in nativescript-material-dialogs, my iOS simulator freezes and I can't do anything. The only thing that is logged is this:
`CONSOLE LOG file: node_modules/nativescript-material-dialogs/dialogs.ios.js:617:0: showUIAlertController <MDCAlertControllerImpl: 0x7fc3e4095e00> Page(27)`

This still works in nativescript-material-dialogs@3.1.12, but anything above that the issue occurs. I mentioned nativescript-material-dialogs@3.1.14 and above in the title because it doesn't look like there's a nativescript-material-dialogs@3.1.13 on npm.

### Is there any code involved? 
`alert('test').then(() => console.log('test'));`

[test-app.zip](https://github.com/Akylas/nativescript-material-components/files/4453197/test-app.zip)

**Comment by farfromrefug** [comment]
It works just fine here. a better log is necessary. Please run on a device to see the real error. Might be related to the runtime

**Comment by kmmccafferty96** [comment]
Hmm that's weird. I tried on my device (iPhone X running iOS 13.4.1) and the same thing happened. Also no logs in the console. I then created a bare-bones nativescript app with only a button that opens a dialog and the same thing happened. I attached the zip of that app here (I removed the node-modules and platforms folders).
[test-app.zip](https://github.com/Akylas/nativescript-material-components/files/4453197/test-app.zip)
